yt/units/tests/test_ytarray.py

```diff
@@ -860,7 +863,10 @@ def binary_ufunc_comparison(ufunc, a, b):
         assert_true(not isinstance(ret, YTArray) and
                     isinstance(ret, np.ndarray))
     if isinstance(ret, tuple):
-        assert_array_equal(ret[0], out)
+        assert isinstance(out, tuple)
+        assert len(out) == len(ret)
+        for o, r in zip(out, ret):
+            assert_array_equal(r, o)
```

Comment:
```suggestion
            assert_array_equal(o, r)
```
just a matter of consistency in ordering.